src/components/doctors.css

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
Optional: Kindly consider adding media queries on the doctors.css file or using any other means in order to make the page responsive on both small and large screen

![Capture2](https://user-images.githubusercontent.com/61447440/191065777-e50672cb-37fb-4a6d-8698-b052cecb1d3c.PNG)

Reply:
@jmoseka 
okay noted will do that